### ISSUE TITLE
metadata test fails

### ISSUE THREAD

**Issue by bandophahita** [ScreenPyHQ/screenpy_selenium#36]
```
       def test_metadata() -> None:
  >       assert __version__.__title__ == "screenpy-selenium"
  E       AssertionError: assert 'screenpy_selenium' == 'screenpy-selenium'
  E         - screenpy-selenium
  E         ?         ^
  E         + screenpy_selenium
  E         ?         ^
```
https://github.com/ScreenPyHQ/screenpy_selenium/actions/runs/5966029789/job/16184798124#step:5:70

I see _why_ it's failing.  I don't understand why it didn't fail a long time ago.

**Comment by bandophahita** [comment]
Well this is weird.  Looking in my local environment 
```
Python 3.11.2 (main, Mar 14 2023, 12:11:57) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from screenpy_selenium import __version__
>>> __version__.__title__
'screenpy-selenium'
```
And yet, another project using `screenpy_selenium` gives me this:
```
Python 3.11.2 (main, Mar 14 2023, 12:11:57) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from screenpy_selenium import __version__
>>> __version__.__title__
'screenpy_selenium'
```

The difference between these two environments is how `screenpy_selenium` package is installed.

**Comment by bandophahita** [comment]
It would appear I needed to update my poetry from 1.5.1 to 1.6.1.  

Using 1.5.1, `poetry build` would create the metadata title as `screenpy_selenium` but when I would use `poetry install` the metadata for the local package would set the title as `screenpy-selenium`.

Moving to 1.6.1 now creates the local package metadata title as `screenpy_selenium`, making it consistent with the build.

However, I still don't exactly know why the change in metadata behavior occurred. I found this [lengthy discussion](https://discuss.python.org/t/change-in-pypi-upload-behavior-intentional-accidental-pebkac/27707/71) talking about how the overall community should proceed but I can't seem to pinpoint a specific issue/PR/commit in poetry that documents the change.

My best guess at the moment is this was a planned change and we simply need to update the test to reflect the change.  I asked for help over in the [poetry discussion](https://github.com/orgs/python-poetry/discussions/8417) in trying to pinpoint the change.
